.coveragerc

```
@@ -5,3 +5,4 @@ omit =
     scipy/setup.py
     scipy/*/setup.py
     scipy/signal/_max_len_seq_inner.py
+disable_warnings = --include is ignored because --source is set (include-ignored)
```

Why not remove `include=*/scipy*` instead of ignoring the warning? Do you want me to do that in gh-14653? If we do want to ignore the warning, I think this should be:
```suggestion
disable_warnings = include-ignored
```

---

Thanks, was not sure about the syntax from the doc.

I would just fix the immediate bug here and clean all these in your PR.